### ISSUE TITLE
Plugin not working with specific linear-gradient background

### ISSUE THREAD

**Issue by arthurmuchir** [seaneking/postcss-hexrgba#6]
There is a problem with following syntax, the color is not transformed :

`background-image: linear-gradient(to top, rgba(#828282, 0.1) 1px, transparent 1px);`

In other situation it works well.

**Comment by seaneking** [comment]
Is it just that specific rgba value, or any value in a `linear-gradient` that doesn't work?

**Comment by arthurmuchir** [comment]
This works well if that is the question :
`linear-gradient(to top, rgba(130, 130, 130, 0.2) 1px, transparent 1px)`

If for test purpose I remove the `1px` that follow the rgba function it works

**Comment by jfurrow** [comment]
I'm getting similar behavior for multiple background declarations, where the second `rgba` call isn't working properly. For example:

Input:
```css
.selector {
  background: linear-gradient(to right, #000 0%, rgba(#000, 0.1) 20%),
    linear-gradient(to left, #000 0%, rgba(#000, 0.1) 20%);
}
```

Output:
```css
.selector {
  background: linear-gradient(to right, 0,0,0 0%, rgba(0,0,0, 0.1) 20%),
    linear-gradient(to left, #000 0%, rgba(#000, 0.1) 20%); /* This should be rgba(0,0,0, 0.1) */
}
```

**Comment by seaneking** [comment]
Fixed in #12
